### PR TITLE
feat(fhir-converter): default date for procedure and immunization

### PR DESCRIPTION
**Pull request by RamilGaripov**
refs. metriport/metriport-internal#2149

### Description
- Default date for `Procedure` and `Immunization` resources

### Testing

- Local
  - [x] Conversion contains the date
  - [x] No issues with inserts

### Release Plan
- [ ] Merge this

### CODE REVIEW

File: packages/fhir-converter/src/templates/cda/Resources/Immunization.hbs [hunk 1]
```diff
@@ -46,6 +46,10 @@
         	"occurrenceDateTime":"{{formatAsDateTime imm.effectiveTime.low.value}}",
         {{else if imm.effectiveTime.value}}
         	"occurrenceDateTime":"{{formatAsDateTime imm.effectiveTime.value}}",
+        {{else if @encounterTimePeriod.low}}
```
Comment (jonahkaye) [inline packages/fhir-converter/src/templates/cda/Resources/Immunization.hbs:49]: why not use 
`            "onsetPeriod":{{>Utils/PeriodOrDefaultPeriod.hbs period=conditionEntry.effectiveTime}},
`

Comment (jonahkaye) [reply packages/fhir-converter/src/templates/cda/Resources/Immunization.hbs:49]: Oh I see no period supported.

Comment (jonahkaye) [reply packages/fhir-converter/src/templates/cda/Resources/Immunization.hbs:49]: I would put High before low in the order of the else if
Sometimes the low is the patients date of birth if the encounter date time is messed up

**Review by jonahkaye — APPROVED** [review]

File: packages/fhir-converter/src/templates/cda/Resources/Immunization.hbs (hunk 1 above)
Comment (RamilGaripov) [reply packages/fhir-converter/src/templates/cda/Resources/Immunization.hbs]: Sounds good!

**Pull request merged**